components/dashboards-web-component/src/utils/WidgetClassRegistry.js

```diff
@@ -65,5 +65,25 @@ global.dashboard = {};
 global.dashboard.registerWidget = function (widgetId, widgetObj) {
     console.warn('[DEPRICATED] \'window.dashboard.registerWidget(widgetId, widgetObj)\' function is depricated. '
                  + 'Instead please use \'Widget.registerWidgetClass(widgetName, widgetClass)\' function.');
+    if (_extendsFromDepricatedWidgetClassVersion(widgetId, widgetObj)) {
+        console.warn('[DEPRICATED] \'@wso2-dashboards/widget@1.0.0\' is depricated. Please user a newer version.');
```

Comment:
Shall we check for the availability of the console object as it is not available in all browsers? ex: IE<10

Reply:
IMO, we don't need to support IE<10. However as per https://developer.mozilla.org/en-US/docs/Web/API/Console/warn `console.warn` is supported in IE8.